lib/tech_for_good_near_you/web/controllers/elm_controller.ex

```diff
@@ -1,7 +1,13 @@
 defmodule TechForGoodNearYou.Web.ElmController do
   use TechForGoodNearYou.Web, :controller
+  alias TechForGoodNearYou.MeetUps
 
   def index(conn, _params) do
     render conn, "index.html"
   end
+
+  def events(conn, _params) do
```

Comment:
I know we're getting the data in the elm app but I think this should probably live in the `EventController` to make it a bit clearer